release-1.8/release_notes_draft.md

```diff
@@ -72,22 +72,43 @@ fundamental aspect of a secure cluster.
   * The`--audit-policy-file` requires `kind` and `apiVersion` fields
     specifying what format version the `Policy` is using.
 
+* The deprecated ThirdPartyResource (TPR) API has been removed.
+  To avoid losing your TPR data, you must
+  [migrate to CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/)
+  before upgrading.
+
+* The following deprecated flags have been removed from `kube-controller-manager`:
+  `replication-controller-lookup-cache-size`, `replicaset-lookup-cache-size`,
+  and `daemonset-lookup-cache-size`. Make sure you no longer attempt to set them.
+
+* StatefulSet: The deprecated `pod.alpha.kubernetes.io/initialized` annotation for
+  interrupting StatefulSet Pod management is now ignored. If you were setting it to
```

Comment:
The annotation can be set to `true` or left blank. However, if it's set to `false`,  previously dormant StatefulSets might become active after upgrading.